### ISSUE TITLE
Transformed data errors

### ISSUE THREAD

**Issue by deepakunni3** [Knowledge-Graph-Hub/kg-covid-19#86]
## zhou_host_proteins

- `biotically interacts with` should be replaced with `biolink:interacts_with`
- Genes with prefix `NCBI` should be replaced with the prefix `NCBIGene`

## ttd

- `T74607` should have a prefix
- `RPGF4_HUMAN` should be remapped to its UniProtKB identifier

```
T17514	biolink:interacts_with	INHBA_HUMAN; INHBB_HUMAN; INHBC_HUMAN; INHBE_HUMAN	RO:0002436	Clinical Trial target
```

The UniProt proteins should be not be multivalued. Each pairwise interaction should be a new line.

##  STRING

- Ensembl proteins should have ENSEMBL prefix

## DrugCentral

- Node and edge CURIE should not be bound in quotes

```
"mefenamic acid"	"mefenamic acid"	biolink:Drug
```

- Invalid node identifier `mefenamic acid` 

```
"https://www.ebi.ac.uk/chembl/compound/inspect/CHEMBL686"	"mefenamic acid"	biolink:Drug
```

- There is another entry in the same file that has mefenamic acid as a `CHEMBL686`
- Also, `https://www.ebi.ac.uk/chembl/compound/inspect/CHEMBL686` should be `CHEMBL.COMPOUND:CHEMBL686`

**Comment by justaddcoffee** [comment]
Thanks @deepakunni3 ! I'll start on these soon

**Comment by justaddcoffee** [comment]
> T74607 should have a prefix

@deepakunni3 quick question - in cases like this where an identifier prefix is not present [here](https://github.com/biolink/biolink-model/blob/master/context.jsonld), what would you recommend we do to stay biolink compliant? Use a full IRI, make up a reasonable prefix (here `TTD:` I guess), something else?

**Comment by deepakunni3** [comment]
Yes, lets go with a TTD prefix. We can revisit later on.

**Comment by justaddcoffee** [comment]
> Ensembl proteins should have ENSEMBL prefix

On this item, in my hands, STRING output seems to already emit protein entries with ENSEMBL prefixes: 

```
id	name	category	description	alias
ENSEMBL:ENSG00000004059	ARF5	biolink:Gene	ADP ribosylation factor 5	NCBIGene:381
ENSEMBL:ENSP00000000233		biolink:Protein		
```

**Comment by saramsey** [comment]
> > T74607 should have a prefix
> 
> @deepakunni3 quick question - in cases like this where an identifier prefix is not present [here](https://github.com/biolink/biolink-model/blob/master/context.jsonld), what would you recommend we do to stay biolink compliant? Use a full IRI, make up a reasonable prefix (here `TTD:` I guess), something else?

Hi all, not to take this issue on a tangent, but just an FYI for the Biolink Model folks, TTD has two different types of identifiers, drugs and targets. The `identifiers.org` registry assigns them two different CURIE prefixes, `ttd.drug` and `ttd.target`. Whatever the Biolink Model folks choose to do, I hope they use two different CURIE prefixes for TTD because otherwise it will make it harder to do IRI expansion.

**Comment by deepakunni3** [comment]
@saramsey Thank you for bringing this to our attention. We will make use of the proper CURIE prefixes.

@justaddcoffee FYI

**Comment by justaddcoffee** [comment]
Thanks @saramsey! This was fixed in #238 (thanks to your sharp eyes)
